### ISSUE TITLE
Punkbuster

### ISSUE THREAD

**Issue by hoesman** [eyza-cod2/zpam3#5]
Hello,

May I bring to attention that "pb_sv_task 0 10 vstr pam_pbsv_cracked" is spamming the pb log every 10 seconds and making it ugly and big for server owners, also why is the cracked check on every client connect self getGuid()? Its not like the server can be changed to crack or original and vice versa mid game.

Regarding the "pbsvuser.cfg" there is also a bunch of outdated/useless "anti-cheat" checks and other stuff that is at the very best of questionable effectiveness today, most of these cvar checks are cheat protected anyways.

There is lots of spam in the log like this for every slot:
Malfunctioning PB Client (slot 3) - Ignoring Cvar Queries

I would suggest to leave just a few basic useful cvar checks of the like:
cl_maxpackets IN 100
com_maxfps IN 125 250
developer IN 0
fx_sort IN 1
r_picmip IN 0 2
r_polygonOffsetScale IN -1 4
r_polygonOffsetBias IN -1 16

**Comment by kratochvilt** [comment]
Hi, 

Thanks for leting me know. Didnt notice that pb log.

That getGuid is executed only untill first player with guid is found. If server is cracked, guid is 0. If original, some value is returned.
Im using it to check if server is cracked or not and print it in readyup.
I also integrated the pb task so if server is original, it should execute pb_sv_guidrelax 0. By default its 7 in the pbsvuser config.
This should basically make sure that you dont have to edit that cvar manually when you running cracked or original server.

But I see its making more problems. Will have to figure out if there is a better way and revert it back eventually.

I found these pb cvars somewhere and thought it would be nice to check everything.
You right that some of the cvars are cheat protected, so its not needed to have them here.
Will consider it as well!

**Comment by eyza-cod2** [comment]
Hi,

Thanks for leting me know. Didnt notice that pb log.

That getGuid is executed only untill first player with guid is found. If server is cracked, guid is 0. If original, some value is returned.
Im using it to check if server is cracked or not and print it in readyup.
I also integrated the pb task so if server is original, it should execute pb_sv_guidrelax 0. By default its 7 in the pbsvuser config.
This should basically make sure that you dont have to edit that cvar manually when you running cracked or original server.

But I see its making more problems. Will have to figure out if there is a better way and revert it back eventually.

I found these pb cvars somewhere and thought it would be nice to check everything.
You right that some of the cvars are cheat protected, so its not needed to have them here.
Will consider it as well!
